ffmpeg.c

```diff
@@ -768,6 +773,65 @@ static int ffmpeg_set_stream(struct ffmpeg *ffmpeg){
 
 }
 
+
+static int alloc_video_buffer(AVFrame *frame, int align)
```

Comment:
Another one of the style consistency things.  Should this function begin with `ffmpeg_`?

Reply:
good point !